platform-core-module/src/main/java/com/softicar/platform/core/module/ajax/listener/BatikCleanerThreadManager.java

```diff
@@ -11,25 +12,28 @@ public class BatikCleanerThreadManager {
 	@SuppressWarnings("deprecation")
 	public void shutdownThread() {
 
-		Log.finfo("starting shutdown of %s", BATIK_CLEANER_THREAD_CLASS);
+		Log.finfo("Starting shutdown of %s", BATIK_CLEANER_THREAD_CLASS);
 		try {
 			Class<?> batikCleanerThreadClass = Class.forName(BATIK_CLEANER_THREAD_CLASS);
 			Field threadField = batikCleanerThreadClass.getDeclaredField("thread");
 			threadField.setAccessible(true);
 			Thread thread = (Thread) threadField.get(null);
 			if (thread != null) {
 				if (new ThreadKiller<>(thread).setTimeout(5000).killAll()) {
-					Log.finfo("successfull shutdown of %s", BATIK_CLEANER_THREAD_CLASS);
+					Log.finfo("Successfull shutdown of %s", BATIK_CLEANER_THREAD_CLASS);
 				} else {
-					Log.finfo("failed shutdown of %s (stupid thread does not want to die)", BATIK_CLEANER_THREAD_CLASS);
-					Log.finfo("calling Thread.stop()");
+					Log.finfo("Failed shutdown of %s (stupid thread does not want to die)", BATIK_CLEANER_THREAD_CLASS);
```

Comment:
I think we should reword `stupid thread does not want to die`

Reply:
ok, removed